### PR TITLE
Spark docker

### PR DESCRIPTION
**Pull request by natsyz**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,6 +23,56 @@ services:
       DOCKER_INFLUXDB_INIT_ORG: eews
       DOCKER_INFLUXDB_INIT_BUCKET: eews
   
+  spark-master:
+    image: cluster-apache-spark:3.4.1
```
Comment (Zsaschz) [inline docker-compose.yml]: any reason for not using the official or managed image like from bitnami?
https://hub.docker.com/r/apache/spark
https://hub.docker.com/r/bitnami/spark

File: dockerfile
Comment (Zsaschz) [inline dockerfile:1]: rename to Dockerfile (with capital D)

**Comment by Zsaschz** [comment]
please move the Dockerfile and start-spark.sh to a separate folder e.g. stream_processing, and all of the spark job into that folder too

File: docker-compose.yml (hunk 1 above)
Comment (natsyz) [reply docker-compose.yml]: i was not aware of that before :(( thank you, i'll revise asap

**Comment by natsyz** [comment]
I changed the docker image to https://hub.docker.com/r/bitnami/spark

**Pull request closed without merging**